tests/unit/component/partitioned_vector_inclusive_scan.cpp

```diff
@@ -0,0 +1,362 @@
+    iota_vector(in, T(1));
+
+    hpx::partitioned_vector<T> out(size, out_policy);
+
+    std::vector<T> ver(in.size());
+    std::iota(ver.begin(), ver.end(), T(1));
+    T val(0);
+
+    hpx::parallel::v1::detail::sequential_inclusive_scan(
+        ver.begin(), ver.end(), ver.begin(), val, opt<T>());
+
+    //sync
+    inclusive_scan_algo_tests_segmented_out_with_policy<T>(
+        size, in_policy, out_policy, in, out, ver, seq);
+    inclusive_scan_algo_tests_segmented_out_with_policy<T>(
+        size, in_policy, out_policy, in, out, ver, par);
+
+    //async
+    inclusive_scan_algo_tests_segmented_out_with_policy_async<T>(
+        size, in_policy, out_policy, in, out, ver, seq(task));
+    inclusive_scan_algo_tests_segmented_out_with_policy_async<T>(
+        size, in_policy, out_policy, in, out, ver, par(task));
+}
+
+template <typename T, typename DistPolicy>
+void inclusive_scan_tests_inplace_with_policy(std::size_t size,
+    DistPolicy const& policy)
+{
+    using namespace hpx::parallel;
+    using hpx::parallel::task;
+
+    // setup verification vector
+    std::vector<T> ver(size);
+    std::iota(ver.begin(), ver.end(), T(1));
+    T val(0);
+
+    hpx::parallel::v1::detail::sequential_inclusive_scan(
+        ver.begin(), ver.end(), ver.begin(), val, opt<T>());
+
+    //sync
+    inclusive_scan_algo_tests_inplace_with_policy<T>(size, policy, ver, seq);
+    inclusive_scan_algo_tests_inplace_with_policy<T>(size, policy, ver, par);
+
+    //async
+    inclusive_scan_algo_tests_inplace_with_policy_async<T>(
+        size, policy, ver, seq(task));
+    inclusive_scan_algo_tests_inplace_with_policy_async<T>(
+        size, policy, ver, par(task));
+}
+
+///////////////////////////////////////////////////////////////////////////////
+
+template <typename T>
+void inclusive_scan_tests()
+{
+    std::size_t const length = 1000000;
+
+    std::vector<hpx::id_type> localities = hpx::find_all_localities();
+
+    inclusive_scan_tests_with_policy<T>(length, hpx::container_layout);
+    inclusive_scan_tests_with_policy<T>(length, hpx::container_layout(3));
+    inclusive_scan_tests_with_policy<T>(length, hpx::container_layout(3, localities));
+    inclusive_scan_tests_with_policy<T>(length, hpx::container_layout(localities));
+ 
+    inclusive_scan_tests_with_policy<T>(1000, hpx::container_layout(1000));
+
+    inclusive_scan_tests_with_policy<T>(1000, hpx::container_layout(1000));
+
+    inclusive_scan_tests_segmented_out_with_policy<T>(length,
+        hpx::container_layout(localities), hpx::container_layout(localities));
+
+    inclusive_scan_tests_segmented_out_with_policy<T>(length,
+        hpx::container_layout(localities), hpx::container_layout(10));
+
+    inclusive_scan_tests_inplace_with_policy<T>(length,
+        hpx::container_layout(localities));
+}
+
+///////////////////////////////////////////////////////////////////////////////
+
+int main()
+{
+    std::vector<hpx::id_type> localities = hpx::find_remote_localities();
```

Comment:
I now understand, the find_remote_localities() call is not necessary here as it is done again in the called functions. Alternatively you may pass `localities` to the actual tests.